packages/py-api/py_api/utilities/parsers.py

```diff
@@ -29,3 +31,16 @@ class AttrDict(dict[Any, Any]):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+def filter_none_values(document: UpdateParticipant) -> Dict[str, Any]:
```

Comment:
The type of document should be BaseModel, because we will use this function in other places with different models.

Reply:
That is a brilliant point, to show that I am very stupid!